src/BookReaderComponent/BookReaderComponent.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * BookReaderTemplate to load BookNavigator components
+ */
+
+import { LitElement, html, css } from 'lit-element';
+
+import '../ItemNavigator/ItemNavigator.js'
+import '../BookNavigator/BookNavigator.js'
+
+export class BookReader extends LitElement {
+
+  static get properties() {
+    return {
+      base64Json: { type: String },
+    };
+  }
+
+  static get styles() {
+    const black = css`#000`;
+    const white = css`#fff`;
+    const grey222 = css`#222`;
+    const grey333 = css`#333`;
+    const grey999 = css`#999`;
+    const primaryCTAFill = css`#194880`;
+    const primaryCTABorder = css`#c5d1df`;
+    const secondaryCTAFill = grey333;
+    const secondaryCTABorder = grey999;
+    const primaryErrorCTAFill = css`#e51c26`;
+    const primaryErrorCTABorder = css`#f8c6c8`;
+
+    return css`
+      :host {
+        display: block;
+        --white: ${white};
+        --blackColor: ${black};
+        --primaryBGColor: ${black};
+        --secondaryBGColor: ${grey222};
+        --tertiaryBGColor: ${grey333};
+        --primaryTextColor: var(--white);
+        --primaryCTAFill: ${primaryCTAFill};
+        --primaryCTABorder: ${primaryCTABorder};
+        --secondaryCTAFill: ${secondaryCTAFill};
+        --secondaryCTABorder: ${secondaryCTABorder};
+        --primaryErrorCTAFill: ${primaryErrorCTAFill};
+        --primaryErrorCTABorder: ${primaryErrorCTABorder};
+        --redBookmarkColor: #eb3223;
+        --blueBookmarkColor: #0023f5;
+        --greenBookmarkColor: #75ef4c;
+      }
+
+      .book-reader {
```

Comment:
Should probably change this to match component name.

```suggestion
      .ia-bookreader {
```